regex-assembly/942180.ra

```diff
@@ -3,27 +3,60 @@
 
 ##!+ i
 
-\d[\"'`]\s+[\"'`]\s+\d
-^admin\s*?[\"'`]
-(?:/\*)+[\"'`]+\s?
-(?:/\*)+[\"'`]+\s?--
-(?:/\*)+[\"'`]+\s?#
-(?:/\*)+[\"'`]+\s?/\*
-(?:/\*)+[\"'`]+\s?{
-[\"'`]\s*?or[\w\s-]+\s*?[+<>=(),-]\s*?[\d\"'`]
-[\"'`]\s*?xor[\w\s-]+\s*?[+<>=(),-]\s*?[\d\"'`]
-[\"'`]\s*?div[\w\s-]+\s*?[+<>=(),-]\s*?[\d\"'`]
-[\"'`]\s*?like[\w\s-]+\s*?[+<>=(),-]\s*?[\d\"'`]
-[\"'`]\s*?between[\w\s-]+\s*?[+<>=(),-]\s*?[\d\"'`]
-[\"'`]\s*?and[\w\s-]+\s*?[+<>=(),-]\s*?[\d\"'`]
-[\"'`]\s*?[^\w\s]?=\s*?[\"'`]
-[\"'`]\W*?[+=]+\W*?[\"'`]
-[\"'`]\s*?[!=|][\d\s!=+-]+.*?[\"'`(].*?$
-[\"'`]\s*?[!=|][\d\s!=]+.*?\d+$
-[\"'`]\s*?like\W+[\w\"'`(]
+##!> define open-comment (?:/\*)
+##!> define quotes [\"'`]
+##!> define quotes-plus-digits [\d\"'`]
+##!> define comment-suffix (?:--|#|{{open-comment}}|\{)
```

Comment:
The last bracket here is escaped, but not on the original regex. Despite this, I think there is no change in the output, right?

Reply:
Yes, that's on purpose. It mustn't be interpreted as a quantifier (`{1}`).